x-pack/plugins/index_management/public/sections/index_list/components/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -462,7 +462,10 @@ class IndexActionsContextMenuUi extends Component {
       <EuiButton
         data-test-subj="indexActionsContextMenuButton"
         iconSide={iconSide}
-        aria-label={`${entity} options`}
+        aria-label={intl.formatMessage({
+          id: 'xpack.idxMgmt.indexActionsMenu.manageButtonAriaLabel',
+          defaultMessage: '{entity} options',
```

Comment:
Since `entry` can be equal `index` or `indices` depending on `this.props.indexNames.length`, let's use plural for this case.

```suggestion
          defaultMessage: '{indexCount, plural, one {index options} other {indices options} }',
        },
        { indexCount: this.props.indexNames.length }
```

Could you please apply the same approach for the similar cases in this file?